codalab/lib/codalab_manager.py

```diff
@@ -248,6 +248,11 @@ def ws_server(self):
         ws_port = self.config['ws-server']['ws_port']
         return f"ws://ws-server:{ws_port}"
 
+    @property  # type: ignore
+    @cached
+    def server_secret(self):
+        return os.getenv("CODALAB_SERVER_SECRET")
```

Comment:
should this be just an env variable? I think we should instead add it as a CodalabArg like the other CODALAB_ env vars

Reply:
This is a CodalabArg. See [here](https://github.com/codalab/codalab-worksheets/blob/use-websockets-for-json-communication/codalab_service.py#L249-L252)

The purpose of that codalab manager property is (like the other properties that already do this) to abstract the env variable away from the WorkerModel (since the CodalabManager passes it in to its constructor). Let me know if this is bad stylistically, though; I certainly wasn't sure about it.